particles/Native/Wasm/source/ServiceParticle.kt

```diff
@@ -1,8 +1,6 @@
 package arcs.test
 
-import arcs.Particle
-import arcs.log
-import arcs.WasmAddress
+import arcs.*
```

Comment:
glob star imports should not pass KtLint

Reply:
Not resolved?